core/events/events.go

```diff
@@ -42,3 +42,19 @@ func NewTextileEvent(bucketname string) TextileEvent {
 		BucketName: bucketname,
 	}
 }
+
+type InvitationStatus string
+
+const (
+	Accepted InvitationStatus = "ACCEPTED"
+	Rejected InvitationStatus = "REJECTED"
+)
+
+type InvitationEvent struct {
+	InviterPublicKey string
+	InvitationID     string
+	Status           InvitationStatus
+	ItemPaths        []string
+	CreatedAt        int64
```

Comment:
Behind the scenes, our invitation metadata could have the Textile message ID and look up the created at and read at fields.